### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by juhuntenburg**
I need to know if any of these files are still used and should be merged

### CODE REVIEW

File: brainwidemap/intro_figure_example_time_series.py [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+
+
+mpl.rcParams.update({'font.size': 10})
+
+one = ONE(base_url='https://openalyx.internationalbrainlab.org',
+          password='international', silent=True)
+
+# save results for plotting here
+pth_res = Path(one.cache_dir, 'brain_wide_map', 'motor_correlates')
```
Comment (juhuntenburg) [inline brainwidemap/intro_figure_example_time_series.py:25]: Let's not write to the ONE cache with intermediate results, you can either have the user set an input or fall back to creating a directory in home

File: brainwidemap/intro_figure_example_time_series.py [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+def get_allen_info():
+    '''
+    Function to load Allen region info, e.g. region colors palette
+    '''
+
+    p = (Path(ibllib.__file__).parent /
+         'atlas/allen_structure_tree.csv')
+
+    dfa = pd.read_csv(p)
```
Comment (juhuntenburg) [inline brainwidemap/intro_figure_example_time_series.py:48]: There is no function to load this in ibllib? Please double check

File: brainwidemap/intro_figure_example_time_series.py [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+    load motion energy for a given session (eid)
+    and video type (e.g. video_type = 'left')
+
+    returns:
+
+    Times: time stamps of motion energy
+    ME: motion energy time series
+    '''
+
+    Times = one.load_dataset(eid, f'alf/_ibl_{video_type}Camera.times.npy',
```
[the remark this answers is not in the document]
Comment (juhuntenburg) [reply brainwidemap/intro_figure_example_time_series.py:106]: also times isn't a great variable name

File: brainwidemap/intro_figure_example_time_series.py [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+
+
+def example_block_structure(eid):
+    '''
+    illustrate block structure in a line plot
+
+    eid = '15f742e1-1043-45c9-9504-f1e8a53c1744'
+    '''
+
+    trials = one.load_object(eid, 'trials')
```
Comment (juhuntenburg) [inline brainwidemap/intro_figure_example_time_series.py:128]: Use SessionLoader

File: brainwidemap/intro_figure_example_time_series.py [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+    '''
+
+    eid = '15f742e1-1043-45c9-9504-f1e8a53c1744'
+    video_type = 'left'
+    # reg = 'PRNr'#'SCiw''MRN'
+    probe = 'probe00'
+    trial_range = [242, 244]
+    nsubplots = 4
+
+    trials = one.load_object(eid, 'trials')
```
Comment (juhuntenburg) [inline brainwidemap/intro_figure_example_time_series.py:152]: Use SessionLoader

File: brainwidemap/intro_figure_example_time_series.py [hunk 6]
```diff
@@ -0,0 +1,334 @@
...
+    tend = trials['intervals'][trial_range[-1]][-1]
+
+    fig, axs = plt.subplots(nsubplots, 1, figsize=(5, 4), sharex=True,
+                            gridspec_kw={'height_ratios': [1, 1, 1, 4]})
+    # plot wheel speed trace
+    # axs.append(plt.subplot(nsubplots,1,1))
+
+    if cnew:
+        Q = []
+        wheel = one.load_object(eid, 'wheel')
```
Comment (juhuntenburg) [inline brainwidemap/intro_figure_example_time_series.py:163]: Use SessionLoader

File: brainwidemap/intro_figure_example_time_series.py [hunk 7]
```diff
@@ -0,0 +1,334 @@
...
+    tstart = trials['intervals'][trial_range[0]][0]
+    tend = trials['intervals'][trial_range[-1]][-1]
+
+    fig, axs = plt.subplots(nsubplots, 1, figsize=(5, 4), sharex=True,
+                            gridspec_kw={'height_ratios': [1, 1, 1, 4]})
+    # plot wheel speed trace
+    # axs.append(plt.subplot(nsubplots,1,1))
+
+    if cnew:
+        Q = []
```
Comment (juhuntenburg) [inline brainwidemap/intro_figure_example_time_series.py:162]: better variable names please

File: brainwidemap/intro_figure_example_time_series.py [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+
+    # neural activity
+    # axs.append(plt.subplot(nsubplots,1,4,sharex=axs[0]))
+
+    if cnew:
+
+        # Load in spikesorting
+        sl = SpikeSortingLoader(eid=eid, pname=probe, one=one, atlas=ba)
+        spikes, clusters, channels = sl.load_spike_sorting()
+        clusters = sl.merge_clusters(spikes, clusters, channels)
```
Comment (juhuntenburg) [inline brainwidemap/intro_figure_example_time_series.py:248]: On purpose we are using all spikes here, not just the good ones?

File: brainwidemap/intro_figure_example_time_series.py [hunk 9]
```diff
@@ -0,0 +1,334 @@
...
+import matplotlib.pyplot as plt
+import pandas as pd
+import matplotlib as mpl
+
+
+
+mpl.rcParams.update({'font.size': 10})
+
+one = ONE(base_url='https://openalyx.internationalbrainlab.org',
+          password='international', silent=True)
```
[the remark this answers is not in the document]
Comment (chris-langfield) [reply brainwidemap/intro_figure_example_time_series.py]: Silent flags removed from `ONE` calls

File: brainwidemap/intro_figure_example_time_series.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (chris-langfield) [reply brainwidemap/intro_figure_example_time_series.py]: All the `ONE` loading methods have been replaced with `SessionLoader` (for comments below as well)

File: brainwidemap/intro_figure_example_time_series.py [hunk 10]
```diff
@@ -0,0 +1,334 @@
...
+    axs[1].axes.yaxis.set_visible(False)
+    # plot licks
+    # axs.append(plt.subplot(nsubplots,1,3,sharex=axs[0]))
+
+    if cnew:
+
+        sess_loader = SessionLoader(one, eid)
+
+        # load DLC
+        sess_loader.load_pose(views=['left', 'right'])
```
[the remark this answers is not in the document]
Comment (chris-langfield) [reply brainwidemap/intro_figure_example_time_series.py]: This is now done by the `SessionLoader` instantiated at the top

File: brainwidemap/motor_correlates/README.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+This code produces panels related to motor correlates of blcok of the brain-wide-map paper. It contains one python script - (fig_motor_correlates_prior.py) - and a npy file (15f742e1-1043-45c9-9504-f1e8a53c1744_left.npy) that contains an example frame being used for illustration in one panel. This file is assumed to be in your current working directory when plotting the paw position on the example frame.
```
[the remark this answers is not in the document]
Comment (juhuntenburg) [reply brainwidemap/motor_correlates/README.md:1]: The commands are already in the main block of that file and the descriptions can be added at the top of that file or somehwere

[the remark this answers is not in the document]
Comment (chris-langfield) [reply brainwidemap/motor_correlates/README.md]: Information from README moved to top of script

File: brainwidemap/intro_figure_example_time_series.py (hunk 2 above)
Comment (chris-langfield) [reply brainwidemap/intro_figure_example_time_series.py]: I checked here and around the `ibllib.atlas` module:

https://int-brain-lab.github.io/iblenv/_modules/ibllib/atlas/regions.html#BrainRegions

It looks like the CSV is opened but the color hex values being manipulated in this code aren't exposed by the `BrainRegions` class. Doesn't look to me like there's a simple way to do it.

**Review by juhuntenburg — COMMENTED** [review]
Not sure why I cannot click approve -- approved

**Pull request merged**